app/views/posts/show.html.erb

```diff
@@ -1 +1,11 @@
-<h1>post show</h1>
+<%= render partial: 'post', locals: { post: @post } %> <br>
+<%= "no comments for this post" if @post.comments.empty?%>
+<div class="comments">
+<% @post.comments.each do |comment|%>
+        <%=render comment %>
```

Comment:
---

Hi @geekelo,

Great job on the progress you've made with your project, especially in adding the views. Your dedication and hard work are commendable. 👏

However, there's a persistent issue from the previous review that still needs to be addressed:

**Change Requested:**
As per the previous feedback, there's an error when trying to access the user's post index page. This issue seems to stem from this line in your code:

```ruby
<%=render comment %>
```

Given that there isn't a `comment` partial (i.e., `_comment.html.erb`) in your views directory, this line will throw an error when there are comments for a post. 

To resolve this, you have a couple of options:
1. **Create a Comment Partial**: If you intend to use a partial for comments, create a `_comment.html.erb` partial in the appropriate views directory and populate it with the necessary content to display each comment.
2. **Directly Display Comments**: If you don't wish to use a partial, you can directly display each comment's content within the loop, replacing the render line.

Please address these concerns and test your view to ensure that the issues are resolved. Keep up the good work, and don't get discouraged. Every challenge is a stepping stone to mastery!

---

Best of luck with your coding journey! 🚀👨‍💻👩‍💻